### PR TITLE
Issue #SB-25388 for SSO in VDN

### PR DESCRIPTION
**Pull request by rajeevsathish**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-25388" title="SB-25388" target="_blank">StorySB-25388</a>  Support for SSO login in Sourcing portal
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_player.env [hunk 1]
```diff
@@ -163,4 +163,7 @@ sunbird_azure_storage_account_name=https://{{sunbird_public_storage_account_name
 
 #Release-4.1.0
 sunbird_google_oauth_ios_clientId={{sunbird_google_oauth_ios_clientId | default("")}}
-sunbird_google_oauth_ios_clientSecret={{sunbird_google_oauth_ios_clientSecret | default("")}}
\ No newline at end of file
+sunbird_google_oauth_ios_clientSecret={{sunbird_google_oauth_ios_clientSecret | default("")}}
+
+#Release-4.2.0
+crypto_encryption_key_external={{crypto_encryption_key_external}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:169]: What’s the default value ? Or it doesn’t require a default is a mandatory variable?

Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sunbird_player.env:169]: @rajeevsathish can you please add comment on what's this variable for

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: ansible/roles/stack-sunbird/templates/sunbird_player.env (hunk 1 above)
Comment (rjshrjndrn) [reply ansible/roles/stack-sunbird/templates/sunbird_player.env:170]: @rajeevsathish encrypt what

**Review by rjshrjndrn — APPROVED** [review]

**Comment by rjshrjndrn** [comment]
LGTM

**Pull request merged**